lib/screens/home/widgets/drawer/list_tile.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class MListTileWidget extends StatelessWidget {
+
+  final Text title;          Text get userTitle =>  title;
+  final IconData icon;       IconData get listTileIcon => icon;
+
```

Comment:
Instead of using the strange listTileTitle function enforce default title value, why not just have a constructor that contains the default values? An example constructor:
```
MListTileWidget({this.title = "about", this. icon = Icons.lightbulb_outline});
```